javalib/src/main/resources/scala-native/process_monitor.cpp

```diff
@@ -1,3 +1,4 @@
+#if defined(SCALANATIVE_CPP_RUNTIME_ENABLED)
```

Comment:
I guess I don't understand. Doesn't this have to also be converted to C? Hard to tell based on C preprocessor format. Perhaps we should go to a 2 space indented style for conditional compilation.

Reply:
This is the implementation we use on Windows. It should be c++ because we use the c++ exceptions there.
The `ifdef` is using the format used on the other c++ files.

Reply:
Yeah, I set up the clang-format so don't have any setup to speak of.

Reply:
I meant the choice between
```c++
#ifdef FOO
```

and

```c++
#if defined(FOO)
```

Reply:
The second one is preferred for more complicated expressions or arguments to `defined`.